sendto_silhouette.py

```diff
@@ -306,6 +306,17 @@ def recursivelyTraverseSvg(self, aNodeList,
         for node in aNodeList:
             # Ignore invisible nodes
             if isinstance(node, BaseElement):
+                # Check if layer name is referring to registration mark or print layer
+                LAYER_LABEL_ATTR_KEY = '{http://www.inkscape.org/namespaces/inkscape}label'
+                if LAYER_LABEL_ATTR_KEY in node.attrib:
+                    layer_label_name = node.attrib[LAYER_LABEL_ATTR_KEY]
```

Comment:
you may access the layer's label through `node.label` property 
see https://gitlab.com/inkscape/extensions/-/blob/master/inkex/elements/_base.py#L535-538

thus simply use
```
if node.layer: